org.nongnu.gsequencer.gsequencer.json

```diff
@@ -0,0 +1,241 @@
+	
+	{
+	    "name": "liblo",
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://download.sf.net/sourceforge/liblo/liblo-0.29.tar.gz",
+		    "sha256": "ace1b4e234091425c150261d1ca7070cece48ee3c228a5612d048116d864c06a"
+		}
+	    ]
+	},      
+	{
+	    "name": "dssi",
+	    "rm-configure": true,
+	    "config-opts": [
+		"--disable-static"
+	    ],
+	    "cleanup": [
+		"/bin"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://download.sf.net/sourceforge/dssi/dssi-1.1.1.tar.gz",
+		    "sha256": "f2c82b073a947c8255284249097667f9b14e660bf86186f3fcd3b3b3e087814e"
+		},
+		{
+		    "type": "script",
+		    "commands": [
+			"autoreconf -fiv"
+		    ],
+		    "dest-filename": "autogen.sh"
+		}
+	    ]
+	},
+
+	{
+	    "name": "audiofile",
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+		    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+		},
+		{
+		    "type": "patch",
+		    "path": "patches/audiofile-gcc6.patch"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "libinstpatch",
+	    "rm-configure": true,
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://downloads.sourceforge.net/project/swami/libinstpatch/1.0.0/libinstpatch-1.0.0.tar.gz",
+		    "sha256": "8943e2226d97609104365f0105be87b326a0b2335cea22d29604a45f8ef1cee7"
+		},
+		{
+		    "type": "patch",
+		    "strip-components": 0,
+		    "path": "patches/instpatch.examples.patch"
+		},
+		{
+		    "type": "script",
+		    "commands": [
+			"autoreconf -fiv"
+		    ],
+		    "dest-filename": "autogen.sh"
+		}
+	    ]
+	},
+	
+	{
+	    "name": "lv2",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"./waf configure --prefix=/app --docdir=/app/share/doc/lv2",
+		"./waf build -j $FLATPAK_BUILDER_N_JOBS",
+		"./waf install"
+	    ],
+	    "cleanup": [
+		"/bin",
+		"/share",
+		"/lib/jack",
+		"/lib/libjackserver.so*"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "http://lv2plug.in/spec/lv2-1.16.0.tar.bz2",
+		    "sha256": "dec3727d7bd34a413a344a820678848e7f657b5c6019a0571c61df76d7bdf1de"
+		}
+	    ]
+	},
+
+	{
+	    "name": "gsequencer",
+	    "config-opts": [
+		"BUILDCC=/usr/bin/gcc -std=gnu99"
+	    ],
+	    "post-install": [
+		"mv /app/bin/gsequencer /app/bin/gsequencer.bin",
+		"mv gsequencer.sh /app/bin/gsequencer",
+		"chmod +x /app/bin/gsequencer",
+		"sed -i -e 's/Icon=gsequencer/Icon=org.nongnu.gsequencer.gsequencer/g' /app/share/applications/gsequencer.desktop",
+		"tar xf chrpath-0.16.tar.gz && cd chrpath-0.16 && ./configure && make",
+		"./chrpath-0.16/chrpath -d /app/lib/gsequencer/libgsequencer.so"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://download-mirror.savannah.gnu.org/releases/gsequencer/2.1.x/gsequencer-2.1.74.tar.gz",
+		    "sha256": "c8d3d1249b45495f60a1985bc75a29d918d1c1c4e752b2fc56a2f2c984595dfd"
+
+		},
+
+		{
+		    "type": "file",
```

Comment:
Why is this file not `archive`?

Reply:
I was looking at VLC and did the same as them.
https://github.com/flathub/org.videolan.VLC

Reply:
It is a bad behavior. Just build it as its own module, have it with `cleanup: ["*"]` and it becomes just a build-only module basically.